spi/common/core-spi/src/main/java/org/eclipse/edc/spi/iam/TokenDecorator.java

```diff
@@ -17,7 +17,10 @@
 /**
  * Implementors of this interface receive the {@link org.eclipse.edc.spi.iam.TokenParameters} instance that is composed
  * when the request travels through the egress.
+ *
+ * @deprecated Will get merged/married with {@code org.eclipse.edc.token.spi.TokenDecorator}.
  */
+@Deprecated
```

Comment:
```suggestion
@Deprecated(since = "0.4.1")
```

Reply:
not really necessary, because it'll get removed before the `refactor/token-handling` branch gets merged into `main`. i @Deprcated it mostly so I don't forget ^^